sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2476,121 +2499,107 @@ object Radar {
                     return
                 }
 
-                sendEvent(customType, metadata, location, events, user, callback)
+                sendLogConversionRequest(conversionType, metadata, callback)
             }
         })
     }
 
     /**
-     * Sends a custom event.
+     * Logs a conversion.
      *
      * @see [](https://radar.com/documentation/api#send-a-custom-event)
      *
-     * @param[customType] The name of the event.
-     * @param[metadata] The metadata associated with the event.
+     * @param[conversionType] The type of the conversion.
+     * @param[metadata] The metadata associated with the conversion.
      * @param[block] A block callback
      */
     @JvmStatic
-    fun sendEvent(
-        customType: String,
-        metadata: JSONObject?,
-        block: (status: RadarStatus, location: Location?, events: Array<RadarEvent>?, user: RadarUser?) -> Unit
+    fun logConversion(
+        conversionType: String,
+        metadata: JSONObject? = null,
+        block: (status: RadarStatus, event: RadarEvent?) -> Unit
     ) {
-        sendEvent(customType, metadata, object : RadarSendEventCallback {
+        logConversion(conversionType, metadata, object : RadarLogConversionCallback {
             override fun onComplete(
                 status: RadarStatus,
-                location: Location?,
-                events: Array<RadarEvent>?,
-                user: RadarUser?
+                event: RadarEvent?
             ) {
-                block(status, location, events, user)
+                block(status, event)
             }
         })
     }
 
     /**
-     * Sends a custom event with a manually provided location.
+     * Logs a conversion with revenue.
      *
      * @see [](https://radar.com/documentation/api#send-a-custom-event)
      *
-     * @param[customType] The name of the event.
-     * @param[location] The location of the event.
-     * @param[metadata] The metadata associated with the event.
+     * @param[conversionType] The type of the conversion.
+     * @param[revenue] The revenue generated by the conversion.
+     * @param[metadata] The metadata associated with the conversion.
      * @param[callback] A callback.
      */
     @JvmStatic
-    fun sendEvent(
-        customType: String,
-        location: Location,
-        metadata: JSONObject?,
-        callback: RadarSendEventCallback
+    fun logConversion(
+        conversionType: String,
+        revenue: Double,
```

Comment:
note: changed revenue from a `Float` to a `Double`. `JSONObject` parses decimal numbers as `Double`s, so it's more consistent to have the client pass in revenue as a `Double` and receive it as a `Double` in the metadata returned in the callback